core-web/libs/edit-content/src/lib/fields/dot-edit-content-binary-field/dot-edit-content-binary-field.component.ts

```diff
@@ -135,14 +136,10 @@ export class DotEditContentBinaryFieldComponent
 
     value: string | null = null;
 
-    @Input({ required: true })
-    set field(contentTypeField: DotCMSContentTypeField) {
-        this.$field.set(contentTypeField);
-    }
     @Input({ required: true }) contentlet: DotCMSContentlet;
     @Input() imageEditor = false;
 
-    $field = signal<DotCMSContentTypeField>({} as DotCMSContentTypeField);
+    $field = input.required<ContentTypeBinaryField>({ alias: 'field' });
```

Comment:
I think we use the old api here because we use this in DOJO. The new api failed.